### PR TITLE
IPA-727. The draft for the Job handling and the folder structure

### PR DESCRIPTION
**Pull request by huukias**
Check for the biggest stupidities.

# Problem / Feature
The folder structure is missing from the task manager and it makes harder to work at the same time.
Also some skeletons for the unit tests and features

# Changes

### CODE REVIEW

File: src/db/storage/sequelizeStorage/Job.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+) => {
+  return db.models.Job.findOne({
+    where: {
+      jobBatchId,
+      chunkId
+    }
+  }).then(toJSON);
+};
+
+const startJob = db => async ({
```
Comment (shirleyqiantingliu) [inline src/db/storage/sequelizeStorage/Job.js]: I think startJob can be in api, but createJob, updateJob, deleteJob will be in db storage.

File: src/db/storage/sequelizeStorage/Job.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  await db.models.Job.update({
+    startedOn: time,
+    progressStatus: PROGRESS_STATUS.IN_PROGRESS
+  }, {
+    where: {
+      jobBatchId,
+      chunkId
+    }
+  });
+  return 200
```
Comment (shirleyqiantingliu) [inline src/db/storage/sequelizeStorage/Job.js]: we should not return an integer here. This file can be refactored.

Comment (huukias) [reply src/db/storage/sequelizeStorage/Job.js]: I completely agree. This is far from finished.

**Comment by huukias** [comment]
Thanks. I will take note of the comments too.

**Pull request merged**